tests/e2e/test_pv_scale_ocs_create_delete_pvcs.py

```diff
@@ -0,0 +1,338 @@
+    test_case = ""
+    timeout = 300
+
+    def pre_req(self):
+        """
+        pre_req def to calculate the median value for the memory leak
+        """
+        log.info(f"Pre-req to eveluate memory leak data "
+                 f"before start of test case")
+        self.median_dict = helpers.memory_leak_median()
+        log.info(f"Median dict values for memory leak {self.median_dict}")
+
+    def create_pvc_pod(self, rbd_sc_obj, cephfs_sc_obj, number_of_pvc, size):
+        """
+        def to create multiple PVC of different type and created pods on them.
+
+        Args:
+            rbd_sc_obj (obj_dict): rbd storageclass object
+            cephfs_sc_obj (obj_dict): cephfs storageclass object
+            number_of_pvc (int): pvc count to be created for each types
+            size (str): size of each pvc to be created eg: '10Gi'
+        """
+        access_modes = [constants.ACCESS_MODE_RWO, constants.ACCESS_MODE_RWX]
+        # Lists to evaluate the pod and pvc created during function call
+        temp_pod_objs, temp_pvc_objs = ([] for i in range(2))
+        future_cephfs_rwo_obj, future_cephfs_rwx_obj = ([] for i in range(2))
+        future_rbd_rwo_obj, future_rbd_rwx_obj = ([] for i in range(2))
+        future_pod_objs, future_temp_pod_objs = ([] for i in range(2))
+        # Flatten to remove the sublist created during future obj
+        flatten = lambda l: [item for sublist in l for item in sublist]
+        self.delete_pod_count = round(number_of_pvc / 2)
+        log.info(f"Inside create_pvc_pod function")
+        # Using Thread to create multiple PVC in parallel
+        # Below code creates pvc with CephFS RWX & RWO and RBD RWX & RWO
+        with ThreadPoolExecutor() as executor:
+            for mode in access_modes:
+                if mode == constants.ACCESS_MODE_RWO:
+                    future_cephfs_rwo_obj.append(
+                        executor.submit(helpers.create_multiple_pvcs,
+                                        sc_name=cephfs_sc_obj.name, namespace=self.namespace,
+                                        number_of_pvc=number_of_pvc, access_mode=mode,
+                                        size=size))
+                else:
+                    future_cephfs_rwx_obj.append(
+                        executor.submit(helpers.create_multiple_pvcs,
+                                        sc_name=cephfs_sc_obj.name, namespace=self.namespace,
+                                        number_of_pvc=number_of_pvc, access_mode=mode,
+                                        size=size))
+        cephfs_rwo_pvcs = [pvc_obj.result() for pvc_obj in future_cephfs_rwo_obj]
+        cephfs_rwo_pvcs = flatten(cephfs_rwo_pvcs)
+        cephfs_rwx_pvcs = [pvc_obj.result() for pvc_obj in future_cephfs_rwx_obj]
+        cephfs_rwx_pvcs = flatten(cephfs_rwx_pvcs)
+        temp_pvc_objs.extend(cephfs_rwo_pvcs)
+        temp_pvc_objs.extend(cephfs_rwx_pvcs)
+        with ThreadPoolExecutor() as executor:
```

Comment:
this is identical code to code above.
What am I missing?

Reply:
soon will provide a patch using functions, which will remove the identical code.

Reply:
added a helper function which can be reused in other scale tests.